src/test/java/com/ksm/wstbackoffice/controller/CountryControllerIT.java

```diff
@@ -54,6 +80,51 @@ public void findAllTest() {
             get("countries").
         then().
             statusCode(200).
-            body("size()", equalTo(5));
+            body("size()", equalTo(21));
+    }
+
+    @ParameterizedTest
+    @MethodSource("findAllByTestArguments")
+    public void findAllByTest(List<String> nameStartsWithFilters, int expectedStatus) {
+        given().
+            contentType(ContentType.JSON).
+            body(nameStartsWithFilters).
```

Comment:
Please read about REST best practices.
Here you use Get with body.

Reply:
done